DuckDuckGo/WindowsBrowserWaitlistView.swift

```diff
@@ -0,0 +1,369 @@
+    let action: (WaitlistViewModel.ViewAction) -> Void
+
+    @State private var shareButtonFrame: CGRect = .zero
+
+    var body: some View {
+        GeometryReader { proxy in
+            ScrollView {
+                VStack(alignment: .center, spacing: 0) {
+                    HeaderView(imageName: "WaitlistInvited", title: UserText.waitlistYoureInvited)
+
+                    Text(UserText.windowsWaitlistInviteScreenSubtitle)
+                        .font(.proximaNova(size: 17))
+                        .foregroundColor(.waitlistText)
+                        .padding(.top, 16)
+                        .lineSpacing(6)
+                        .fixedSize(horizontal: false, vertical: true)
+
+                    Text(UserText.waitlistInviteScreenStepTitle(step: 1))
+                        .font(.proximaNova(size: 17, weight: .bold))
+                        .foregroundColor(.waitlistText)
+                        .padding(.top, 28)
+                        .padding(.bottom, 8)
+
+                    Text(UserText.windowsWaitlistInviteScreenStep1Description)
+                        .font(.proximaNova(size: 17))
+                        .foregroundColor(.waitlistText)
+                        .lineSpacing(6)
+
+                    if #available(iOS 14.0, *) {
+                        Text(AppUrls().windowsBrowserDownloadURL.absoluteString.dropping(prefix: "https://"))
+                            .font(.proximaNova(size: 17, weight: .bold))
+                            .foregroundColor(.waitlistBlue)
+                            .menuController(UserText.waitlistCopy) {
+                                action(.copyDownloadURLToPasteboard)
+                            }
+                            .scaledToFit()
+                    } else {
+                        Text(AppUrls().windowsBrowserDownloadURL.absoluteString.dropping(prefix: "https://"))
+                            .font(.proximaNova(size: 17, weight: .bold))
+                            .foregroundColor(.waitlistBlue)
+                            .padding(.top, 6)
+                    }
+
+                    Text(UserText.waitlistInviteScreenStepTitle(step: 2))
+                        .font(.proximaNova(size: 17, weight: .bold))
+                        .foregroundColor(.waitlistText)
+                        .padding(.top, 22)
+                        .padding(.bottom, 8)
+
+                    Text(UserText.windowsWaitlistInviteScreenStep2Description)
+                        .font(.proximaNova(size: 17))
+                        .foregroundColor(.waitlistText)
+                        .lineSpacing(6)
+
+                    if #available(iOS 14.0, *) {
+                        InviteCodeView(title: UserText.waitlistInviteCode, inviteCode: inviteCode)
+                            .menuController(UserText.waitlistCopy) {
+                                action(.copyInviteCodeToPasteboard)
+                            }
+                            .padding(.top, 28)
```

Comment:
I was still seeing sizing issues on the Menu Controller view, it's possible to somewhat work around them this way:

```suggestion
                        InviteCodeView(title: UserText.waitlistInviteCode, inviteCode: inviteCode)
                            .menuController(UserText.waitlistCopy) {
                                action(.copyInviteCodeToPasteboard)
                            }
                            .fixedSize()
                            .padding(.top, 28)
```

This approach will fix the sizing issue, but the view will no longer be vertically centered.